src/components/ProductsList/ProductsList.tsx

```diff
@@ -4,16 +4,38 @@ import { ProductCard } from '../ProductCard/ProductCard';
 
 import styles from './styles.module.scss';
 import { Product } from '@/types/Product';
+import { DeviceCategory } from '@/enums';
 
 interface Props {
   paginationOfDevice: Product[];
+  category: DeviceCategory;
 }
 
-export const ProductsList: React.FC<Props> = ({ paginationOfDevice }) => {
+export const ProductsList: React.FC<Props> = ({
+  paginationOfDevice,
+  category,
+}) => {
+  const getProductPath = (itemId: string) => {
+    switch (category) {
+      case DeviceCategory.PHONES:
+        return `/phones/${itemId}`;
+      case DeviceCategory.TABLETS:
+        return `/tablets/${itemId}`;
+      case DeviceCategory.ACCESSORIES:
+        return `/accessories/${itemId}`;
+      default:
+        return '/';
+    }
+  };
+
   return (
     <article className={styles.phones_list}>
       {paginationOfDevice.map(phone => (
```

Comment:
it's better to name it like "device", not `phone`

Also we have const `sortedPhones ` in `ProductsCatalog`. In fact, it can include other devices too, so we need to rename it. (It's not your changes, but you can fix it)